components/NumberDisplay.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { FC } from 'react';
+interface Props {
+  title: string;
+  value: string;
+}
+
+const NumberDisplay: FC<Props> = ({ title, value }) => {
+  return (
+    <div className="flex-1 my-2 p-4 text-center bg-ocf-gray-900 rounded-2xl dark:bg-ocf-gray-800">
```

Comment:
I think the color here needs to be slightly darker to match the designs, and I actually just added it to main as `ocf-gray-1000`. Don't worry too much about `dark:` prefixes in styles, since we'll be writing for only dark mode right now it's implicit.
```suggestion
    <div className="flex-1 my-2 p-4 text-center bg-ocf-gray-1000 rounded-2xl">
```